lib/repl.js

```diff
@@ -852,6 +852,57 @@ ArrayStream.prototype.resume = function() {};
 ArrayStream.prototype.write = function() {};
 
 const requireRE = /\brequire\s*\(['"](([\w@./-]+\/)?(?:[\w@./-]*))/;
+
+
+const fsTabbableMethods = [
+  'access',
+  'accessSync',
+  'chmod',
+  'chmodSync',
+  'chown',
+  'chownSync',
+  'createReadStream',
+  'createWriteStream',
+  'exists',
+  'existsSync',
+  'lchmod',
+  'lchmodSync',
+  'lchown',
+  'lchownSync',
+  'lstat',
+  'lstatSync',
+  'mkdir',
+  'mkdirSync',
+  'open',
+  'openSync',
+  'readdir',
+  'readdirSync',
+  'readFile',
+  'readFileSync',
+  'readlink',
+  'readlinkSync',
+  'realpath',
+  'realpath',
+  'realpathSync',
+  'realpathSync',
+  'rmdir',
+  'rmdirSync',
+  'stat',
+  'statSync',
+  'symlink',
+  'symlinkSync',
+  'truncate',
+  'truncateSync',
+  'unlink',
+  'unlinkSync',
+  'utimes',
+  'utimesSync',
+].join('|');
+
+const fsRE = new RegExp(
+  `fs\\.(?:${fsTabbableMethods})\\(["'](([\\w@./-]+\\/)?(?:[\\w@./-]*))`
```

Comment:
Does it need the `fs.` restriction?

Reply:
the restriction is not needed, but its there to avoid unexpected collisions for functions with similar names.

```javascript
function readFile(fileName) {
}
readFile('...<Tab> // this will work and might be unexpected for users
```

this is my opinion, what do you think?

Reply:
Do you think people are _really_ likely to <Tab> in the repl during an existing line-statement when they don't want autocomplete?

Reply:
I personally agree with @Fishrock123 and think it is better to keep this open. A false positive does not hurt here out of my perspective.